assembl/static2/js/app/integration/101/components/button101/button101.jsx

```diff
@@ -0,0 +1,39 @@
+// @flow
+import React from 'react';
+import { Button } from 'react-bootstrap';
+
+type Props = {
+  /** Optional possible values: any strings */
+  label?: string,
+  /** Optional possible types: 'info', 'danger' */
+  type?: string,
```

Comment:
you can use `type?: 'info' | 'danger',`

Reply:
@cedricmessiant corrected, thanks for the tips